chore(hm-instances): Add page about available HM instances (#1951)
mendersoftware/mender-docs: 10.General/00.Hosted Mender regions/docs.md

---
title: Overview
taxonomy:
    category: docs
---

> ender is available in multiple regions you can choose from, depending on your data processing and compliance needs. Each instance is independent of the others and provides all the Mender features.

Reviewer: ```suggestion
Hosted Mender is available in multiple regions you can choose from, depending on your data processing and compliance needs. Each instance is independent of the others and provides all the Mender features.
```

Author: Done, thanks!

## Regions

The list of the currently supported regions follows:

* US instance - `https://hosted.mender.io`
* EU instance - `https://eu.hosted.mender.io` 

> The EU instance is compliant with General Data Protection Regulation (GDPR).

Reviewer: ```suggestion
The EU instance is compliant with the General Data Protection Regulation (GDPR).
```

Author: Done, thanks!

---
title: Overview
taxonomy:
    category: docs
---

> Hosted Mender provides various domains to connect to, depending on what are your data processing and compliance needs.

Reviewer: ```suggestion
Mender is available in multiple regions you can choose from, depending on your data processing and compliance needs. Each instance is independent of the others and provides all the Mender features.
```

## Regions

> Currently Mender has two instances:

Reviewer: ```suggestion
The list of the currently supported regions follows:
```

Author: done!

* US instance - `https://hosted.mender.io`
* EU instance - `https://eu.hosted.mender.io` 

> EU instance is compliant with General Data Protection Regulation,(GDPR).

Reviewer: ```suggestion
The EU instance is compliant with General Data Protection Regulation (GDPR).
```

Author: done!

> 
> Both instances are fully independent.

Reviewer: ```suggestion
```

## Region selection

When setting up your account, you can choose between available Hosted Mender instances:

> ![region selector](region-selector.png)

Reviewer: ```suggestion
![region selector](region-selection.png)
```

Author: done!

## Switching between instances

> // TODO: is there an instruction about switching instances?

Author: done!
